v03_pipeline/lib/paths.py

```diff
@@ -198,6 +216,24 @@ def sex_check_table_path(
     )
 
 
+def valid_filters_path(
+    dataset_type: DatasetType,
+    sample_type: SampleType,
+    callset_path: str,
+) -> str | None:
+    if (
+        not Env.EXPECT_WES_FILTERS
+        or not dataset_type.expect_filters(sample_type)
+        or 'part_one_outputs' not in callset_path
```

Comment:
what are 'part_one_outputs' and 'part_two_outputs'?

Reply:
@matren395 can give some context here!  I think it's just that internal exome callsets have always been delivered in two different folders, and we run an extra step to join the `filters` part onto the first part!

Reply:
That's what I'd believe this is too, yes!

WES callsets are delivered in two parts usually, with two folders for Part_One_Outputs and Part_Two_Outputs. Part_One_Outputs are the "normal" sharded VCFs, one set of sharded VCFs per chromosome, complete with tons of annotations and information. Part_Two_Outputs is I believe just one VCF per chromosome, containing the sites to filter the Part_One VCFs to .